src/SnowflakeTransformation.php

```diff
@@ -50,33 +51,65 @@ public function __construct(Config $config, LoggerInterface $logger)
     public function createManifestMetadata(
         array $tableNames,
         ManifestManager $manifestManager,
-        bool $transformationFailed = false
+        bool $usingLegacyManifest,
+        bool $transformationFailed = false,
     ): void {
         $tableStructures = $this->getTables($tableNames, $transformationFailed);
         foreach ($tableStructures as $tableDef) {
             $columnsMetadata = (object) [];
+            $schema = [];
+            $primaryKeysNames = $tableDef->getPrimaryKeysNames();
+
             /** @var SnowflakeColumn $column */
             foreach ($tableDef->getColumnsDefinitions() as $column) {
                 $columnsMetadata->{$column->getColumnName()} = $column->getColumnDefinition()->toMetadata();
+
+                $dataTypes = [
+                    'base' => [
+                        'type' => $column->getColumnDefinition()->getBasetype(),
+                    ],
+                    'snowflake' => [
```

Comment:
Tady bych klidně použil konstantu `Snowflake::METADATA_BACKEND`

Reply:
To je určitě lepší, použil jsem